app/controllers/admin/customers_controller.rb

```diff
@@ -66,7 +66,12 @@ def collection
       return Customer.where("1=0") unless json_request? && params[:enterprise_id].present?
 
       Customer.of(managed_enterprise_id).
-        includes(:bill_address, :ship_address, user: :credit_cards)
+        includes(:bill_address, :ship_address, user: :credit_cards).
+        joins(:orders).
+        merge(Spree::Order.complete.not_state(:canceled)).
```

Comment:
For the record:

![Screenshot from 2020-12-16 12-52-21](https://user-images.githubusercontent.com/762088/102345362-92c91880-3f9d-11eb-9f59-133606a1da69.png)

Reply:
Confirmed. There's no order in state `returned` in any of the triumvirate instances.

Reply:
alright! all states covered! `returned` however might need more work in the future because return authorizations don't cause a partial refund AFAIK so we can't adjust the customer balance accordingly. That's the best we can do without touching that other feature.